speechbrain/lobes/augment.py

```diff
@@ -529,3 +531,60 @@ def _prepare_csv(folder, filelist, csv_file, max_length=None):
                         )
     finally:
         sb.utils.distributed.ddp_barrier()
+
+
+class AddNoiseSimple(torch.nn.Module):
```

Comment:
More context needed on what this is. Especially since it introduces a new dependency on turtle ?

Reply:
mistake, was trying some stuff out and forgot to exclude

Reply:
done